src/types/commands.ts

```diff
@@ -624,6 +624,13 @@ export interface DuplicatePivotCommand {
   newPivotId: string;
 }
 
+export interface DuplicatePivotInNewSheetCommand {
+  type: "DUPLICATE_PIVOT_IN_NEW_SHEET";
+  pivotId: UID;
+  newPivotId: UID;
+  newSheetId: UID;
+}
+
```

Comment:
Move it in the section below (near INSERT_NEW_PIVOT_COMMAND), this section is for the core commands